### PR TITLE
Adding the clean FeedController with the corresponding endpoint

### PR DESCRIPTION
**Pull request by Nde-Dilan**

### CODE REVIEW

File: src/main/java/com/socialize/service/entityService/FeedService.java
Comment (NganeEmmanuel) [inline src/main/java/com/socialize/service/entityService/FeedService.java:1]: * use ```fianals``` and add the @RequireAllArgsConstructor on this class. then remove the ```@Autowire``` annotation
* e,g ``` private final UserRepository userRepository```
* then uses:
```
@Service
@RequireAllArgsConstructor
public class FeedService {
    //implementation
}
```

field injections are not advisable in so when we use the @Allargs constructor from lombok it automatically does the construction with parameters of all final fields in the class, which is a method injection and that is the recommended way. so fix this

File: src/main/java/com/socialize/work-of-dilan.js
Comment (NganeEmmanuel) [inline src/main/java/com/socialize/work-of-dilan.js:1]: not sure why this file is here please remove it

File: src/main/java/com/socialize/model/User.java
[the remark this answers is not in the document]
Comment (Nde-Dilan) [reply src/main/java/com/socialize/model/User.java:1]: An Error, sorry 👍 ;)

File: src/main/java/com/socialize/dto/PostDTO.java
[the remark this answers is not in the document]
Comment (Nde-Dilan) [reply src/main/java/com/socialize/dto/PostDTO.java:1]: Another error , I was still trying to understand the code. sorry.

File: src/main/java/com/socialize/service/entityService/FeedService.java
Comment (Nde-Dilan) [reply src/main/java/com/socialize/service/entityService/FeedService.java:1]: Okay, Understood 🫡.

File: src/main/java/com/socialize/work-of-dilan.js
Comment (Nde-Dilan) [reply src/main/java/com/socialize/work-of-dilan.js:1]: Sorry, it was just to track what i needed to do.
Will remove it

File: src/main/resources/application.properties
[the remark this answers is not in the document]
Comment (Nde-Dilan) [reply src/main/resources/application.properties:1]: Understood 🫡

File: src/main/java/com/socialize/service/entityService/UserServiceImpl.java
[the remark this answers is not in the document]
Comment (Nde-Dilan) [reply src/main/java/com/socialize/service/entityService/UserServiceImpl.java:1]: Okay, I changed it back.

**Review by NganeEmmanuel — APPROVED** [review]

**Pull request merged**